src/Chapter2.hs

```diff
@@ -741,8 +755,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains x = filter (\l-> x `elem` l)
```

Comment:
If you use the prefix notation for the `elem` function, you can notice an opportunity for eta-reduction here 🙂 
```suggestion
contains x = filter (elem x)
```

Reply:
thanks for the tip (: